.golangci.yml

```diff
@@ -16,3 +16,6 @@ run:
     - vpc/bpf/filter
     - vpc/bpf2
     - vpc/limitsold
+issues:
+  exclude:
+    - Using the variable on range scope .* in function literal
```

Comment:
This has caught a number of real issue (primarily around launching goroutines in functions). I'd rather not exclude this.